beetsplug/embyupdate.py

```diff
@@ -39,9 +39,7 @@ def api_url(host, port, endpoint):
         hostname_list.insert(0, "http://")
         hostname = "".join(hostname_list)
 
-    joined = urljoin(
-        "{hostname}:{port}".format(hostname=hostname, port=port), endpoint
-    )
+    joined = urljoin(f"{hostname}:{port}", endpoint)
```

Comment:
While this is a URL we're formatting too, I think using `str.format` would be completely unnecessary here.  The actual variable names are perfect and this is just much shorter.